Clinical-Genomics/cgweb
# Bootstrap upgrade

How to test:
0. Install on stage
1. Open OP-stage in a browser-window 
2. Open OP-prod in another browser-window
3. Compare the two browsers windows
4. Go through every OP view and compare
Expected outcome: 
Every view looks exactly the same
